applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS
+	bool "DVFS module"
+	depends on SOC_NRF54H20_CPUAPP
+	depends on BT_MAX_CONN=1
+	default y
+	help
+	  This option enable DVFS module which switches frequency and voltage according
+	  to application needs.
+
+if DESKTOP_DVFS
+
+config DESKTOP_DVFS_RETRY_TIMEOUT_MS
+	int "Retry timeout"
+	default 50
+	range 1 1000
+	help
+	  Timeout in milliseconds for retrying DVFS operation.
```

Comment:
The module requests DVFS frequency change on application events. It's possible that subsequent request is performed while the frequency change triggered by the previous request is still in progress. In that case, module needs to retry DVFS frequency change after some time.

The option specifies timeout for retrying the DVFS frequency change.